app/models/gemnasium_client.rb

```diff
@@ -22,6 +22,28 @@ def all_projects_and_counts
     advisories
   end
 
+  # rubocop:disable MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def all_dependencies
+    gems = {}
+    slugs = all_projects
+    slugs.shuffle.each do |slug|
```

Comment:
shuffle was a weird artifact of the original code before it was used in sulreports.  Removing and shortifying.